expyfun/stimuli/_tracker.py

```diff
@@ -738,6 +739,16 @@ def __init__(self, trackers, max_lag=1, rand=None):
         self._tracker_history = np.array([], dtype=int)
         self._response_history = np.array([], dtype=int)
         self._x_history = np.array([], dtype=float)
+        
+        self._dealer_id = id(self)
+        self._callback('dealer_identity', json.dumps(dict(
+            dealer_id=self._dealer_id,
+            tracker_type='dealer')))
```

Comment:
``TrackerDealer`` isn't a tracker, so I wouldn't even include the ``tracker_type`` keyword arg. There are many kinds of trackers, but only one dealer.